packages/tizen_audio_manager/example/lib/volume_control.dart

```diff
@@ -0,0 +1,123 @@
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:tizen_audio_manager/tizen_audio_manager.dart';
+
+/// A widget with audio controllers.
+class VolumeControlScreen extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _VolumeControlScreenState();
+}
+
+class _VolumeControlScreenState extends State<VolumeControlScreen> {
+  AudioVolumeType _currentPlaybackType = AudioVolumeType.none;
+  AudioVolumeType _selectedType = AudioVolumeType.ringtone;
+  int _currentVolume = 0;
+  int _maxVolume = 0;
+  Timer? _timer;
+  VolumeChangedEvent? _volumeChangedEvent;
+  StreamSubscription<VolumeChangedEvent>? _subscription;
+  final List<DropdownMenuItem<AudioVolumeType>> _dropdownButtonItems =
+      AudioVolumeType.values
+          .where((AudioVolumeType e) => e != AudioVolumeType.none)
+          .map((AudioVolumeType e) => DropdownMenuItem<AudioVolumeType>(
+              value: e, child: Text(e.toString().split('.').last)))
+          .toList();
+
+  @override
+  void initState() {
+    super.initState();
+    _onAudioTypeChanged(_selectedType);
+
+    _timer = Timer.periodic(const Duration(seconds: 1), (Timer timer) async {
+      final AudioVolumeType type =
+          await AudioManager.volumeController.currentPlaybackType;
+
+      if (type != _currentPlaybackType) {
+        setState(() {
+          _currentPlaybackType = type;
+        });
+      }
+    });
+
+    _subscription = AudioManager.volumeController.onChanged
+        .listen((VolumeChangedEvent event) {
+      setState(() {
+        _volumeChangedEvent = event;
+
+        if (event.type == _selectedType) {
+          _currentVolume = event.level;
+        }
+      });
+    });
+  }
+
+  @override
+  void dispose() {
+    _timer?.cancel();
+    _subscription?.cancel();
+    super.dispose();
+  }
+
+  Future<void> _onAudioTypeChanged(AudioVolumeType? type) async {
+    type = type ?? AudioVolumeType.ringtone;
+    final int currentVolume =
+        await AudioManager.volumeController.getLevel(type);
+    final int maxVolume = await AudioManager.volumeController.getMaxLevel(type);
+
+    setState(() {
+      _selectedType = type!;
+      _currentVolume = currentVolume;
+      _maxVolume = maxVolume;
+    });
+  }
+
+  void _onVolumeSliderChanged(double value) {
+    AudioManager.volumeController.setLevel(_selectedType, value.toInt());
+
+    setState(() {
+      _currentVolume = value.toInt();
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Volume control')),
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          Text('Current playback type: ' +
+              _currentPlaybackType.toString().split('.').last),
```

Comment:
Prefer adjacent string concatenation.
```suggestion
          Text('Current playback type: ${_currentPlaybackType.name}'),
```